contrib/docker/Dockerfile.apache

```diff
@@ -4,21 +4,21 @@ FROM php:7.4-apache-buster
 COPY contrib/docker/php.production.ini "$PHP_INI_DIR/php.ini"
 
 # Install Composer
-ENV COMPOSER_VERSION 1.9.2
+ENV COMPOSER_VERSION 1.10.6
 ENV COMPOSER_HOME /var/www/.composer
+ENV PATH="~/.composer/vendor/bin:./vendor/bin:${PATH}"
+COPY . /var/www/
```

Comment:
Best practice: I'd still leave the `COPY` command at the end of the whole image setup `RUN` command. Followed by the 
So code changes in the filesystem don't trigger the whole image rebuild.

Followed by another single RUN command of the remaining composer/code build commands

Reply:
thanks, i updated the commit accordingly.